.mockery.yaml

```
@@ -0,0 +1,387 @@
+issue-845-fix: True
+with-expecter: True
+packages:
+  github.com/onflow/flow-go/network/p2p:
+    config:
+      dir: "./network/mocknetwork"
+      outpkg: "mocknetwork"
+      filename: "{{.InterfaceName}}.go"
```

we use snake case for filenames. I think this is the source of the conflicting mock files. please adjust the filename field, remove all of the new mock files created in this PR, then regenerate.
```suggestion
      filename: "{{ .InterfaceName | snakecase }}.go"
```